src/index.js

```diff
@@ -23,33 +23,43 @@ import {
 
 import { initCSSPolyfill } from "./scroll-timeline-css"
 
-initCSSPolyfill();
+export function initPolyfill() {
+  if (initCSSPolyfill()) {
+    return true;
```

Comment:
This early return skips the checks below.

Reply:
initCSSPolyfill returns true iff the browser natively supports SDA.  So in that case, the js polyfill (below) is also going to be skipped.  

I will add documentation to describe this.